blockkit/elements.py

```diff
@@ -717,3 +719,27 @@ def __init__(
         indent: Optional[int] = None,
     ):
         super().__init__(elements=elements, style=style, indent=indent)
+
+class FileInput(FocusableElement):
+    type: str = "file_input"
+    filetypes: Optional[List[str]] = Field(None)
+    maxfiles: Optional[int] = Field(None)
+
+    def __init__(
+        self,
+        *,
+        action_id: Optional[str] = None,
+        filetypes: Optional[List[str]] = None,
+        maxfiles: Optional[int] = None,
+        focus_on_load: Optional[bool] = None,
```

Comment:
According to the [Slack Docs](https://api.slack.com/reference/block-kit/block-elements#file_input) `file_input` element doesn't support `focus_on_load`.